### DISCUSSION TITLE
River in Google Colab

### DISCUSSION THREAD

**Discussion by DEV-3D** [online-ml/river#696] [Q&A] [answered]
Hello,
I was using river in colab untill yesterday without any issues.  Today I cant import it using "from river import drift" due to "ImportError: numpy.core.multiarray failed to import"

![image](https://user-images.githubusercontent.com/44900031/130975856-de7df1b6-407a-4f25-9500-f931e9a0a243.png)

I already uninstalled and reinstalled numpy and river but no success so far.

How can I fix it?

Thanks

**Comment by MaxHalford** [comment]
Mmm can you try doing `pip install river==0.7.0` and try again?

**Reply by DEV-3D** [reply]
i just did the downgrade:  same problem!
I am very confused because it was working perfectly yesterday.
Problem always the same:

![image](https://user-images.githubusercontent.com/44900031/130982011-74e0b6fc-83f0-4e97-87ad-fe90849b7b95.png)

![image](https://user-images.githubusercontent.com/44900031/130982172-911bfe58-7cb3-4268-9f4d-edb57fb776ce.png)

![image](https://user-images.githubusercontent.com/44900031/130982228-58823fcd-e808-4e08-bd47-07c962e7b9b5.png)

Using jupyter, locally, is working just fine.  Maybe is something in google's end.

**Reply by MaxHalford** [reply]
Maybe try to downgrade numpy and restart the notebook?

**Comment by giturra** [comment, marked as answer]
I got the same problem today but it was solved running this command:

`pip install -U numpy ` 

[link to stackoverflow question](https://stackoverflow.com/questions/20518632/importerror-numpy-core-multiarray-failed-to-import)
